showcase/app/templates/page-components/form/super-select.hbs

```diff
@@ -362,6 +382,7 @@
         as |F|
       >
         <F.Label>Lorem ipsum dolor</F.Label>
+        {{! @glint-expect-error }}
```

Comment:
I added a note to the existing ticket: https://hashicorp.atlassian.net/browse/HDS-5090

But SuperSelect would be another good opportunity for generics because when you render an option some of the examples access properties on the option which throws a type error.